### ISSUE TITLE
Matches are sometimes created with 0 length

### ISSUE THREAD

**Issue by thiderman** [drunkenfall/drunkenfall#126]
During the last tournament we had some issues where Match.Length was erroneously set to zero in the latter tryouts. This happened in the matches after the initial four, so something is probably borky with the additional ones.
This doesn't affect gameplay, but does break the judging interface.

I wrote a unit test:

```
func TestAllMatchesHaveALengthSet(t *testing.T) {
	assert := assert.New(t)
	for x := 0; x < 1000; x++ {
		tm := testTournament(27)
		err := tm.StartTournament()
		assert.Nil(err)

		for _, m := range tm.Tryouts {
			assert.Equal(m.Length, 10)
		}

		for _, m := range tm.Semis {
			assert.Equal(m.Length, 10)
		}

		assert.Equal(tm.Final.Length, 20)
	}
}
```

But it runs fine for the 1000 iterations without breaking. Something else is wrong.

One way of solving this would be to update the juding interface to default to a length of 10 if there is something off with the data, but maybe that should not be required.

**Comment by thiderman** [comment]
This is somewhat fixed in #133, but only because that starts to use `match.end` rather than `match.length`. This means that the frontend has a special case in terms of length of matches. It's definitely not the end of the world, but it's also not super cool.

**Comment by thiderman** [comment]
Until this happens again, I choose not to care.
